metaflow/plugins/cards/ui/src/global.css

```diff
@@ -31,119 +31,15 @@
     "Fira Mono", "Droid Sans Mono", "Courier New", monospace;
 }
 
-* {
-  box-sizing: border-box;
-}
-
-html,
-body {
-  background: var(--bg);
-  color: var(--black);
-  font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
-    "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  font-size: 14px;
-  font-weight: 400;
-  line-height: 1.5;
-  text-size-adjust: 100%;
+html, body {
```

Comment:
why not just apply these to `.card` as well and remove global.css?

Reply:
These remove extra padding around the `.card` when it is within our page. The same rules are in `.card` too for when we are embedded.